### PR TITLE
update roadmap with 2.14

### PR DESCRIPTION
**Pull request by zroubalik**
https://github.com/kedacore/keda/issues/4710

### CODE REVIEW

**Review by JorTurFer — COMMENTED** [review]
2023?

**Comment by zroubalik** [comment]
> 2023?

😅 It has been a long day, thanks!

File: ROADMAP.md [hunk 1]
```diff
@@ -14,14 +14,15 @@ Here is an overview of our current release estimations:
 
 | Version | Estimated Release Date                               |
 |:--------|:-----------------------------------------------------|
-| v2.11   | June 13th, 2023                                      |
 | v2.12   | September 12th, 2023                                 |
 | v2.13   | January 11th, 2023 *(planned later due to holidays)* |
+| v2.14   | April 12th, 2023                                     |
```
Comment (zroubalik) [inline ROADMAP.md:19]:
```suggestion
| v2.13   | January 11th, 2024 *(planned later due to holidays)* |
| v2.14   | April 12th, 2024                                     |
```

**Comment by zroubalik** [comment]
> > 2023?
> 
> 😅 It has been a long day, thanks!

@JorTurFer fixed, v2.13 was also wrong 😄

**Pull request merged**